docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+|  **read_length_r1**            | read_groups |                                                           |  Length of sequencing reads in `file_r1`; this corresponds to the number of sequencing cycles of the first end.                                                                                                                                                                                                                                                              |                      |  
+|  **read_length_r2**             | read_groups |                                                           |  Length of sequencing reads in `file_r2`; this corresponds to the number of sequencing cycles of the second end.                                                                                                                                                                                                                                                             |                      |  
+|  **insert_size**                | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  For paired-end sequencing, the average size of sequences between two sequencing ends. Required only for paired-end sequencing.                                                                                                                                                                                                                                              |                      |  
+|  **sample_barcode**             | read_groups |                                                           |  According to the SAM specification, this is the expected barcode bases as read by the sequencing machine in the absence of errors.                                                                                                                                                                                                                                          |                      |  
+|  **library_name**               | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of a sequencing library made from a molecular sample or a sample pool (multiplex sequencing).                                                                                                                                                                                                                                                                          |                      |  
+|  **fileName**       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of the file.                                                                            |                      |  
```

Comment:
For `Permissible values`, add
- String
- String values must meet the regular expression: `^[A-Za-z0-9_\\.\\-\\[\\]\\(\\)]+$`